jenkins/scripts/integration_clean.sh

```diff
@@ -49,9 +49,9 @@ function cleanup() {
   echo "Old leftover resources are cleaned successfully!"
 }
 
-# Run feature tests and main, master and release* integration tests in the Frankfurt region
-if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || \
-   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]]
+# Run feature tests, e2e tests, main, master and release* tests in the Frankfurt region
+if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"* ]] || \
+   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]] || [[ "${UPDATED_BRANCH}" == "release"* ]]
```

Comment:
same

Reply:
line 52-54 is not necessary. I have changes them in #273, I have deleted them and add cleanup separately for FRA1

Reply:
@mboukhalfa ^^

Reply:
Please dont add anyting to this file. Because it is running once in a day for cleanup